windows-installer/docs/INSTALL.md

```diff
@@ -0,0 +1,85 @@
+
+In terms of authenticator plugins, `standalone`, `manual` and `webroot` are supported. DNS plugins will be available soon.
+
+For now, no installer plugins are available. `apache`, `nginx` will be available soon, and a new installer for IIS is under development.
+
+Certificate automated renewals are supported.
+
+All options available for a given supported command will work as described by the documentation in Certbot for Windows.
+
+## Installation
+
+The following operations needs to be executed using an account with **administrative privileges** on the machine you want to install Certbot.
+
+Certbot can be installed through an installer that is available:
+- on GitHub: go to the Certbot GitHub release (https://github.com/certbot/certbot/releases), select the last version of Certbot (v0.40.0 currently), and download the installer named `certbot-beta-installer-win32.exe in the assets.
+- on Certbot official website: _[specific URL and procedure to be defined]_
+
+The Certbot development team ensures the authenticity of the Windows installer only for these two sources.
+
+Once downloaded, double-click on the installer to run it. You will need to select the location for Certbot installation directory, or keep the default one (`C:\Program Files(x86)\Certbot).
+
+At this point Certbot is ready to run.
+
+## Using Certbot
+
+Certbot is a command line interface with no UI. To interact with it you will need a command line: the classic one (`cmd.exe`) and Powershell aor both supported. You can also use Bash for Windows (available if you installed GIT for Windows for instance) if you prefer.
+
+The command line needs to be executed under **administrative privileges**. Otherwise Certbot will fail to run.
+
+Once the command line is opened, simply write the following command to get the Certbot help:
+```
+certbot --help
+```
+
+Indeed the Certbot installer set up the `PATH` to expose the command line interface `certbot`. So you can run it from any folder, or even include it in your scripts.
+
+In respect with the supported features as described in the _Supported features_ section, you can check the inline help or https://certbot.eff.org/docs/ to check how to use a specific feature in Certbot.
+
+## Configuration
+
+Certbot works essentially with a configuration folder, where all the configuration files and generated certificates are available.
+
+On Windows, this folder is `C:\Certbot`. This can be changed using the `--config-dir` flag on the Certbot CLI.
+
+Within this folder you will find the following folders:
+* `C:\Certbot\live`: contains current versions of issued certificates and keys
+* `C:\Certbot\archive`: contains all current and previous versions of issued certificates and keys
+* `C:\Certificate\logs`: contains Certbot activity logs
+
+Doing backups of `C:\Certbot` folder is everything required to save your Certbot installation.
```

Comment:
is all that's required

Reply:
Maybe also note that `archive` is sensitive because it contains cryptographic private keys, which should not be disclosed to unauthorized parties?